core/tests/protractor_mobile/navigation.js

```diff
@@ -142,7 +142,7 @@ describe('Navigation features on mobile', function() {
         oppiaLogo, 'Could not click oppia logo');
       oppiaLogo.click();
       waitFor.pageToFullyLoad();
-      expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/splash');
```

Comment:
Maybe change the test message to `'should navigate to Home page by clicking on the Oppia logo`

Reply:
Changed it here and in the desktop tests.
Though, I don't see the need for doing this since we are still calling the home page as splash page, right? I mean, these changes are merely done to ensure that the URL doesn't contain the 'splash' word, everything else should stay the same, right?

Reply:
URL "/" is the homepage and at the homepage URL we are showing splash page whereas earlier the home page "/" was redirected to "/splash" page. (In this test we are testing the URL which is the homepage URL and earlier it was a splash page URL)

Reply:
Yup, that makes sense. Thanks for explaining!